.github/workflows/cppcheck.yml

```diff
@@ -0,0 +1,29 @@
+name: Cppcheck
+on: [push, pull_request]
+
+jobs:
+  cppcheck:
+    name: Cppcheck
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install Cppcheck
+        run: |
+          sudo apt update
+          sudo apt install -y cppcheck
+          sudo apt install -y python3 python3-pip zip wget
+          pip3 install pygments
+          wget https://raw.githubusercontent.com/danmar/cppcheck/main/htmlreport/cppcheck-htmlreport
```

Comment:
Who is this `danmar` and how stable is this url?

Reply:
This is official Cppcheck repository on Github - https://github.com/danmar/cppcheck/
As I see in Github history, this file is in this place 12 years(was never moved)